File: 0705-design-hashset/0705-design-hashset.java
class MyHashSet {
    List<Integer>l;
    public MyHashSet() {
        l=new ArrayList<>();
    }
    
    public void add(int key) {
        if(!l.contains(key))
        l.add(key);
    }
    
    public void remove(int key) {
        for(int i=0;i<l.size();i++)
        {
            if(l.get(i)==key)
            l.remove(i);
        }
    }
    
    public boolean contains(int key) {
        for(int i=0;i<l.size();i++){
            if(l.get(i)==key)
            return true;
        }
        return false;
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */

File: 0705-design-hashset/0705-design-hashset.py
class MyHashSet:

    def __init__(self):
        self.head = ListNode(val=None)
        self.tail = self.head

    def add(self, key: int) -> None:
        if self.contains(key):
            return
        self.tail.next = ListNode(val=key)
        self.tail = self.tail.next

    def remove(self, key: int) -> None:
        node = self.head
        while node:
            if node.next and node.next.val == key:
                node.next = node.next.next
                if node.next == None:
                    self.tail = node
                break
            node = node.next

    def contains(self, key: int) -> bool:
        node = self.head
        while node:
            if node.val == key:
                return True
            node = node.next
        return False
        
class ListNode:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)